config/settings/base.py

```diff
@@ -366,7 +366,12 @@
 OPENSEARCH_INDEX_PREFIX = env("OPENSEARCH_INDEX_PREFIX")
 
 OPENSEARCH_DSL = {
-    "default": {"hosts": env("OPENSEARCH_HOST")},
+    "default": {
```

Comment:
why change `OPENSEARCH_HOSTS` now?

Reply:
https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/
it was experimenting, to check